tasks/deploy/config/mainnet/morpho-blue-usde-dai-182day.ts

```diff
@@ -0,0 +1,126 @@
+import { Address, encodeAbiParameters, parseEther, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    MAINNET_DAI_ADDRESS,
+    SIX_MONTHS,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { MAINNET_FACTORY_NAME } from "./factory";
+import { MAINNET_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const MAINNET_MORPHO_BLUE_USDE_DAI_182DAY_NAME =
+    "ElementDAO 182 Day Morpho Blue USDe/DAI Hyperdrive";
+
+const CONTRIBUTION = parseEther("100");
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken:
+                "0x4c9EDD5852cd905f086C759E8383e09bff1E68B3" as `0x${string}`, // Mainnet sUSDe
+            oracle: "0xaE4750d0813B5E37A51f7629beedd72AF1f9cA35" as `0x${string}`,
+            irm: "0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const MAINNET_MORPHO_BLUE_USDE_DAI_182DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: MAINNET_MORPHO_BLUE_USDE_DAI_182DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: toBytes32(
+            MAINNET_MORPHO_BLUE_USDE_DAI_182DAY_NAME.slice(0, 32),
+        ),
+        salt: toBytes32("0x4202"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.065"),
```

Comment:
I was just looking at the 7day rate again and i think i told you wrong. it should be 7.5% instead of 6.5%